### ISSUE TITLE
Basic auth not working

### ISSUE THREAD

**Issue by zetiv** [codazoda/nolific#3]
Tried to open home page and popup is opening with username/password. Looking thru the code i see that you are loading users from 
users.ini file but that file is missing

**Comment by ragn0708** [comment]
I've even tried added a file called users.ini, containing 

     PHP_AUTH_USER=name
     PHP_AUTH_PW=password

with no success. All ideas welcome.

**Comment by codazoda** [comment]
I've added a default username/password and pushed that change. This isn't the best way to solve this problem, but it solves it somewhat quickly. I did this because the project is getting some activity on Hacker News this morning. Here are the commits.

The work to add admin, admin as the default username/password.

https://github.com/codazoda/nolific/commit/ad0ae7032a8177d9ce3f25ca0c0ad26536e4d576

The work to mention the defaults in the readme.

https://github.com/codazoda/nolific/commit/9e90b44131d78eed7395120257abc33e3bca70ff
